tests/measurements/test_expval.py

```diff
@@ -93,3 +135,25 @@ def test_shape_shot_vector(self, obs):
         shot_vector = (1, 2, 3)
         dev = qml.device("default.qubit", wires=3, shots=shot_vector)
         assert res.shape(dev) == (len(shot_vector),)
+
+    @pytest.mark.parametrize("shots", [None, 1000, [1000, 10000]])
+    def test_projector_var(self, shots, mocker):
+        """Tests that the variance of a ``Projector`` object is computed correctly."""
+        dev = qml.device("default.qubit", wires=3, shots=shots)
```

Comment:
Function name and docstring needs to reflect `expval` instead of `var`.

Reply:
Done!